apps/supernova/appProps.template.json

```diff
@@ -2,11 +2,11 @@
   "endpoint": "https://myalertmanager.com/api/v2",
   "filterLabels": [],
   "initialFilters": {},
-  "predefinedFilters": {},
+  "predefinedFilters": [],
   "silenceExcludedLabels": [],
   "silenceTemplates": [],
   "theme": "theme-dark",
   "username": "Jane Doe",
   "enableHashedRouting": false,
-  "baseUrl": "/"
+  "basePath": "/"
```

Comment:
We might need to document this into the readme of supernova

Reply:
They documented here:
https://github.com/cloudoperators/juno/blob/main/apps/supernova/README.md#configuration